demisto_sdk/commands/validate/validators/PB_validators/PB131_is_silent_playbook_relationships.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.content_graph.common import ContentType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.trigger import Trigger
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Playbook, Trigger]
+
+
+class IsSilentPlaybookRelationshipsValidator(BaseValidator[ContentTypes]):
+    error_code = "PB131"
+    description = "A silent-Playbook/Trigger must point to a silent-Playbook/Trigger"
+    rationale = "A silent-Playbook/Trigger must point to a silent-Playbook/Trigger"
+    error_message = (
+        "Your silent-Playbook/Trigger does not point to a silent-Playbook/Trigger"
+    )
+    related_field = ""
+    is_auto_fixable = False
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message,
```

Comment:
```suggestion
    ) -> List[ValidationResult]:
        def get_types_for_err_msg(c: ContentItem) -> tuple[ContentTypes, ContentTypes]:
            if isinstance(c, Trigger):
                return ContentTypes.TRIGGER, ContentTypes.PLAYBOOK
            return ContentTypes.PLAYBOOK, ContentTypes.TRIGGER
        return [
            ValidationResult(
                validator=self,
                message=self.error_message.format(*get_types_for_err_msg(content_item)),
```
just to make the error message a bit nicer - please verify the change